README.md

```diff
@@ -88,14 +88,32 @@ ALE can be installed from its update site: http://www.kermeta.org/ale-lang/updat
 
 ## Contributing
 
+<details>
+  <summary><b>Requirements</b></summary>
+
+  - [Maven 3.x](https://maven.apache.org/download.cgi)
+  - [Java 8 JDK](https://adoptopenjdk.net/upstream.html)
+  - [Eclipse IDE for DSL Developers](https://www.eclipse.org/downloads/packages/) (latest release)
```

Comment:
I would have appreciated that we also indicate in the readme that the GEMOC studio is a good start for building ALE, (The studio is one of the primary use of ALE and ALE was made to integrate in it.

Reply:
What do you mean "for building ALE"? If someone just wants to enhance ALE then fetching the whole studio would clearly be overkill. What about a new section after *Contributing*, e.g.:

```
## Integration with the GEMOC Studio

ALE is part of the GEMOC Studio, an IDE for the development, integration, and use of heterogeneous executable modeling languages. As such, it can be used to define the semantics of domain-specific languages:

 - learn more about the GEMOC Studio: http://gemoc.org/studio.html
 - check the GEMOC Studio on GitHub: https://github.com/eclipse/gemoc-studio
 - check the GEMOC ALE engine on GitHub: https://github.com/eclipse/gemoc-studio-execution-ale
```

Reply:
The GEMOC Studio is more than that from ALE point of view: It also provides the _"Model Debug"_ capability while the basic ALE launch configuration provides only the _"Model Run"_ capability.
In addition, contributor are supposed to know that if they change ALE API there's a high risk of breaking the  support for debug of model with semantics written in ALE.

I admit that _Eclipse IDE for DSL Developers_ is enough for some simple scenarios(as user), but I think we should promote more advanced use and make sure contribution take this into account. (and thus encourage coherence between these projects)

So I would say that _Eclipse IDE for DSL Developers_ package  is a minimal requirement while _GEMOC Studio_ package is the recommanded requirement

Reply:
Indeed, you made very important points here which are about both using of ALE and contributing to the language.

> The GEMOC Studio [...] provides the "Model Debug" capability while the basic ALE launch configuration provides only the "Model Run" capability. [...] we should promote more advanced use.

I agree. That's about _using_ ALE. So what about a new item in the **Main features** section:

```md
## Main features

 - **Powerful debug/animation environment**: its integration in the [GEMOC Studio](https://gemoc.org) 
   provides a powerful debug environment featuring a debug trace and a Sirius animator, 
   see [Integration with the GEMOC Studio](#integration-with-the-gemoc-studio) for more details.
```

(I'd like to put "DSL" somewhere but didn't find an adequate wording yet) and a new section before **Contributing**:

```md
## Integration with the GEMOC Studio

ALE is part of the GEMOC Studio, an IDE for the development, integration, and use of heterogeneous 
executable modeling languages. Within the Studio, ALE can be used to define the semantics of 
domain-specific languages and is strengthened with debugging and animation features. 
See the following links to learn more about the GEMOC Studio and ALE:
```

> contributors are supposed to know that if they change ALE API there's a high risk of breaking the support for debug of model with semantics written in ALE. [...] I think we should make sure contributions take this into account (and thus encourage coherence between these projects)

I fully agree. That's about _contributing_ to ALE. We should add this to the README so that contributors are aware of the risk of changing public API. However, I see no point in encouraging the use of the GEMOC Studio as an IDE for developing ALE.

Reply:
As per our discussion, using the GEMOC Studio as an IDE is actually useful because it makes possible to quickly test whether the plug-ins (e.g. ALE + GEMOC's ALE engine) behave as expected in the Studio. I'm still puzzled as to why it never causes incompatibilities/conflicts between the version of the plug-ins in the workspace and the version of the same plug-ins installed in the Studio, but since experience shows that this setup works we should outline that in the README.